core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -499,23 +504,28 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
       e = assertThrows(classOf[ExecutionException], () => alterResult.values.get(topic2).get,
         () => s"$desc: Expect InvalidPartitionsException when requesting a noop")
       assertTrue(e.getCause.isInstanceOf[InvalidPartitionsException], desc)
-      assertEquals("Topic already has 3 partitions.", e.getCause.getMessage, desc)
+      exceptionMsgStr = if (TestInfoUtils.isKRaft(testInfo)) {
+        "Topic already has 3 partition(s)."
+      } else {
+        "Topic already has 3 partitions."
```

Comment:
I wonder if we can unify the error messages? The differences do not seem interesting.

Reply:
There are too many error messages in the `ReplicationControl` layer, and I'm not sure how many other layers need to be changed. I think this can be a different PR to keep the messages intact if required.

Reply:
Fair enough. I don't think asserting the exact messages is all that useful to be honest, but not need to change that here.